oops/hosts/juno/jiram/__init__.py

```
@@ -97,37 +97,26 @@ class JIRAM(object):
 
     #===========================================================================
     @staticmethod
-    def initialize(ck='reconstructed', planets=None, asof=None,
-                   spk='reconstructed', gapfill=True,
-                   mst_pck=True, irregulars=True):
-        """Initialize key information about the JIRAM instrument.
+    def initialize(asof=None, **kwargs):
```

Ruff (0.15.20)

[warning] Missing return type annotation for staticmethod `initialize`

Add return type annotation: `None`

(ANN205)